OpenEdXMobile/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -257,6 +260,22 @@ public void showNoNetworkDialog() {
     }
 
     @Override
+    public boolean dispatchTouchEvent(MotionEvent ev) {
+        Animation errorMessageAnim = activityLoginBinding.errorLayout.getAnimation();
+        if (errorMessageAnim == null || errorMessageAnim.hasEnded()) {
+            ViewAnimationUtil.hideMessageBar(activityLoginBinding.errorLayout);
+        }
+        return super.dispatchTouchEvent(ev);
+    }
+
+    // make sure that on the login activity, all errors show up as a dialog as opposed to a flying snackbar
+    @Override
+    public boolean showErrorMessage(String header, String message) {
+        super.showErrorDialog(header, message);
```

Comment:
It is public, and I worry that showErrorMessage will be called polymorphicaly. I did it just to be sure that we are showing the dialog every time.

What do you think, should I get rid of it, or leave it just for safety's sake until this is redone?